### PR TITLE
More logging

### PR DESCRIPTION
**Pull request by cole-h**
Avoid cluttering logs further.

Also, split out the `info!` with a newline to make logging more consistent.

### CODE REVIEW

File: ofborg/src/tasks/eval/nixpkgs.rs [hunk 1]
```diff
@@ -568,6 +568,14 @@ impl<'a> EvaluationStrategy for NixpkgsStrategy<'a> {
 fn request_reviews(maint: &maintainers::ImpactedMaintainers, pull: &hubcaps::pulls::PullRequest) {
     if maint.maintainers().len() < 10 {
         for maintainer in maint.maintainers() {
+            if let Ok(pr) = pull.get() {
```
Comment (grahamc) [inline ofborg/src/tasks/eval/nixpkgs.rs]: Let's move that `pull.get()` out of this loop :)

Comment (cole-h) [reply ofborg/src/tasks/eval/nixpkgs.rs]: Whoops, you're right.

File: ofborg/src/tasks/build.rs [hunk 2]
```diff
@@ -373,7 +373,8 @@ impl notifyworker::SimpleNotifyWorker for BuildWorker {
         };
 
         info!("ok built ({:?}), building", status);
-        info!("Lines:\n-----8<-----");
+        info!("Lines:");
+        info!("-----8<-----");
```
Comment (grahamc) [inline ofborg/src/tasks/build.rs:377]: Nice. Once we're sloggin', we could even pass the lines in as a context argument!

**Comment by grahamc** [comment]
Thanks!

**Pull request merged**